### PR TITLE
doc: small documentation review

### PR DESCRIPTION
**Pull request by ldez**
### Description

- fix syntax coloration.
- same language, same code fence.
- fix typo.
- add some magic.

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -161,34 +161,34 @@ As seen in the previous example, you can combine multiple rules.
...
-    [frontends.frontend2.routes.test_1]
-    rule = "Host:test1.localhost,test2.localhost"
-  [frontends.frontend3]
-  backend = "backend2"
-    [frontends.frontend3.routes.test_1]
-    rule = "Path:/test1,/test2"
+[frontends.frontend2]
+   [frontends.frontend2.routes.test_1]
+   rule = "Host:test1.localhost,test2.localhost"
+ [frontends.frontend3]
```
Comment (timoreimann) [inline docs/basics.md]: Extra leading space?

File: docs/user-guide/kubernetes.md [hunk 2]
```diff
@@ -562,7 +562,11 @@ could set `disablePassHostHeaders` to true in your toml file and then enable pas
...
 ## Excluding an ingress from Træfik
-You can control which ingress Træfik cares about by using the "kubernetes.io/ingress.class"
+
+You can control which ingress Træfik cares about by using the `kubernetes.io/ingress.class`
 annotation. By default if the annotation is not set at all Træfik will include the
 ingress. If the annotation is set to anything other than traefik or a blank string
 Træfik will ignore it.
+
+
+![](http://i.giphy.com/ujUdrdpX7Ok5W.gif)
```
Comment (timoreimann) [inline docs/user-guide/kubernetes.md]: Not sure why, but still ☺️

Comment (ldez) [inline docs/user-guide/kubernetes.md]: See https://docs.traefik.io/user-guide/swarm/#access-to-your-apps-through-trfk 😉

Comment (timoreimann) [inline docs/user-guide/kubernetes.md]: 🤣

**Pull request merged**